### DISCUSSION TITLE
§ shortcut for G14

### DISCUSSION THREAD

**Discussion by MatyasDoktor** [seerge/g-helper#1884] [Q&A] [answered]
I would like to bind the § symbol to Fn + C as a custom key bind as I use it often when typing.

Does anyone know how what I need to enter as the custom command to achieve this? Thanks!

**Comment by seerge** [comment]
I don't know if that symbol has a unique keycode (most probably not), but why don't you check Sharkeys app if it offers a binding for it ?

**Reply by MatyasDoktor** [reply, marked as answer]
I tried that and was unable to get it to work. I did however get it to work with windows power toys!
